core/src/main/java/org/elasticsearch/index/IndexSettings.java

```diff
@@ -127,6 +128,16 @@
     public static final Setting<Integer> MAX_SLICES_PER_SCROLL = Setting.intSetting("index.max_slices_per_scroll",
         1024, 1, Property.Dynamic, Property.IndexScope);
 
+    /**
+     * The number of shard copies required to be active before returning from an index creation operation.
+     */
+    public static final Setting<ActiveShardCount> WAIT_FOR_ACTIVE_SHARDS_SETTING =
+        new Setting<>("index.wait_for_active_shards",
+                         (String) null,
```

Comment:
I changed it to "1", though I don't think it would simplify the `resolve` code nor the `parseString` code.  I can remove the null check in `parseString` if I also pass in defaults to all the rest actions `request.param("wait_for_active_shards")`, though I'm not sure what that buys us?